### DISCUSSION TITLE
Allow editing for provisioned alert rules with flag similar to dashboards 'allowUiUpdates'

### DISCUSSION THREAD

**Discussion by lmuhlha** [grafana/grafana#54259] [Alerting]
We are migrating all of our alerts to a new Grafana + datasource set up and are unable to migrate alerts via api while still allowing users to make UI updates afterwards.

**Comment by lmuhlha** [comment]
I just found this similar thread: https://github.com/grafana/grafana/discussions/53490
But I wonder if you have any advice for teams migrating alerts and dashboards to a new set up for their users. Telling everyone who wants to be able to edit via the UI in the future to re-make everything manually is not an option for us.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
